.ci/cirrus.runner.yml

```
@@ -7,11 +7,11 @@ task:
         XCODE_VERSIONS: "16.1"
         DISK_SIZE: 190
       - MACOS_VERSION: sequoia
-        XCODE_VERSIONS: "16.4,\"26-rc\",16.3,16.2,16.1,16"
+        XCODE_VERSIONS: "26,16.4,26,16.3,16.2,16.1,16"
         ADDITIONAL_IOS_BUILDS: "18.5,18.4,18.2,17.5"
         DISK_SIZE: 400
```

Could you please also add the MetalToolchain to Sequoia?

```suggestion
        ADDITIONAL_IOS_BUILDS: "18.5,18.4,18.2,17.5"
        XCODE_COMPONENTS: "\"MetalToolchain\""
        DISK_SIZE: 400
```